### PR TITLE
Create LIT5886MonthsLength.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Calendaric note in DSEthiop2, to be upgraded

### CODE REVIEW

File: new/LIT5886MonthsLength.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5886MonthsLength" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">MonthsLength</title>
```
Comment (thea-m) [inline new/LIT5886MonthsLength.xml:8]: This is the title which will be displayed, could you give it the one you would like?

File: new/LIT5886MonthsLength.xml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-04-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT5886MonthsLength" passive=""/>
```
Comment (thea-m) [inline new/LIT5886MonthsLength.xml:57]: Could you add the manuscript ID to @passive? But actually, the relation is not really necessary, the ID will be used in the manuscript record

File: new/LIT5886MonthsLength.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT5886MonthsLength.xml:8]: Done

File: new/LIT5886MonthsLength.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT5886MonthsLength.xml:57]: Done

**Comment by thea-m** [comment]
I think you have carried out these changes in https://github.com/BetaMasaheft/Works/pull/97/files, so I will close this PR and we will only keep https://github.com/BetaMasaheft/Works/pull/97/files.

**Pull request closed without merging**